### PR TITLE
feat(textarea): add tokens and update shim

### PR DESCRIPTION
**Pull request by mindarelus**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/angular/src/forms/styles/_properties.forms.scss [hunk 1]
```diff
@@ -65,6 +65,10 @@
           forms-variables.$clr-forms-invalid-color,
           $lightness: 30%
         )};
+      --clr-forms-textarea-padding: forms-variables.$clr-forms-baseline +
+        variables.$clr_baselineRem_2px
+        forms-variables.$clr-forms-baseline *
+        2;
```
Comment (valentin-mladenov) [inline projects/angular/src/forms/styles/_properties.forms.scss:71]:
```suggestion
      --clr-forms-textarea-padding: #{forms-variables.$clr-forms-baseline} +
        #{variables.$clr_baselineRem_2px}
        #{forms-variables.$clr-forms-baseline} *
        2;
```

Consider my suggestion.

File: projects/angular/src/forms/styles/_textarea.clarity.scss [hunk 2]
```diff
@@ -47,8 +45,19 @@
...
+      border-radius,
+      clr-forms-textarea-border-radius,
+      forms-variables.$clr-forms-baseline * 0.5,
+      variables.$clr-use-custom-properties
+    );
+    @include mixins.css-var(
+      padding,
+      clr-forms-textarea-padding,
+      $textarea-vertical-padding $textarea-horizontal-padding,
+      variables.$clr-use-custom-properties
```
Comment (valentin-mladenov) [inline projects/angular/src/forms/styles/_textarea.clarity.scss:58]: The variables are not needed any more use their values directly. Brackets are optional.

```suggestion
    (forms-variables.$clr-forms-baseline + variables.$clr_baselineRem_2px) (forms-variables.$clr-forms-baseline * 2),
      variables.$clr-use-custom-properties
```

File: projects/angular/src/forms/styles/_textarea.clarity.scss [hunk 3]
```diff
@@ -47,8 +45,19 @@
...
+    );
+
     font-size: forms-variables.$clr-forms-field-font-size;
 
     &:focus {
```
Comment (valentin-mladenov) [inline projects/angular/src/forms/styles/_textarea.clarity.scss:63]: Here you have to calculate selected status from figma using border color `--cds-alias-object-interaction-background-highlight`

**Review by valentin-mladenov — CHANGES_REQUESTED** [review]
Check my comments.

File: projects/angular/src/forms/styles/_textarea.clarity.scss [hunk 4]
```diff
@@ -10,20 +10,33 @@
 
 @include mixins.exports('forms.textarea') {
   // @TODO Fix selectors when old forms are removed to allow better defaults
+  .clr-form-horizontal {
+    .clr-textarea-wrapper {
+      // margin-top: 0;
```
Comment (dtsanevmw) [inline projects/angular/src/forms/styles/_textarea.clarity.scss:15]: That's needed for textarea in horizontal forms.

**Review by dtsanevmw — CHANGES_REQUESTED** [review]
Revert the deletion of `margin-top` for the horizontal state and it seems that the text area is now misaligned and the subtext starts before it 
Screenshot 2023-08-01 at 11 14 39

**Review by dtsanevmw — APPROVED** [review]

**Review by valentin-mladenov — APPROVED** [review]
LGTM

**Pull request merged**